firebase-app-distribution/src/main/java/com/google/firebase/app/distribution/NewReleaseFetcher.java

```diff
@@ -186,7 +194,8 @@ private long getInstalledAppVersionCode(Context context) throws FirebaseAppDistr
     try {
       pInfo = context.getPackageManager().getPackageInfo(context.getPackageName(), 0);
     } catch (PackageManager.NameNotFoundException e) {
```

Comment:
Thoughts on extracting the getPackageInfo logic into a method so we could DRY up this code?

Reply:
I have made the change to see how it looks. It adds some overall code, but does make it look nicer.

Reply:
I think you could also add the try/catch statements to the new method so we don't need to duplicate that code